packages/driver/cypress/e2e/e2e/origin/commands/aliasing.cy.ts

```diff
@@ -17,14 +17,14 @@ context('cy.origin aliasing', () => {
 
     it('fails for dom elements outside origin', (done) => {
       cy.on('fail', (err) => {
-        expect(err.message).to.equal('`cy.get()` could not find a registered alias for: `@welcome_button`.\nYou have not aliased anything yet.')
+        expect(err.message).to.equal('`cy.get()` could not find a registered alias for: `@link`.\nYou have not aliased anything yet.')
         done()
       })
 
-      cy.get('[data-cy="welcome"]').as('welcome_button')
+      cy.get('[data-cy="cross-origin-secondary-link"]').as('link')
```

Comment:
I removed the element with `data-cy="welcome"` since it wasn't referenced elsewhere, so I had to update this test.